.github/workflows/test.yml

```diff
@@ -93,3 +93,19 @@ jobs:
             echo "does not match"
             exit 1
           fi
+      - name: "Test call"
+        id: test-call
+        uses: ./
+        with:
+          module: github.com/shykes/daggerverse/hello@v0.3.0
+          call: hello
+      - name: "Test call (check)"
```

Comment:
This test case is in the `version` job - just for clarity, either it should be in a different job, or we could rename this job to something else?

Reply:
Yeah I can rename it. I didn't realize the workflow was called version, I just saw test.yml